_data/diversity.yaml

```
@@ -0,0 +1,38 @@
+
+    - title: "Mentorización de ponentes"
+      description: "Para facilitar que gente de cualquier transfondo pueda hacer una charla, estamos organizando un grupo de voluntarios con experiencia dando charlas para mentorizar a gente que nunca haya dado una charla, priorizando a los integrantes de grupos infrarrepresentados. Si te interesa utilizar esta ayuda, contacta con nosotros: hola@vlctechfest.org."
+
+    - title: "¿Crees qué podemos hacer algo más?"
+      description: "Si se te ocurre alguna otra acción que podamos llevar a cabo para mejorar la inclusividad, no dudes en compartirla con nosotros: hola@vlctechfest.org"
+
+en:
+  title: Diversity
+  description: "In VLCTechHub we include the diversity in our statutes and we are always looking for new initiatives to integrate the infrarrepresented groups in the technical community. Here you can find the different initiatives we do for this festival:"
+  actions:
+    - title: "Code of conduct"
+      description: "To ensure the event goes without incidents and where everybody is welcome, we include a code of conduct. In the case it is necessary to report a violation of the code of conduct, you can get in contact with Emma (emma@vlctechhub.org) or David (david@vlctechhub.org)."
+
+    - title: "Mentoring of speakers"
+      description: "To facilitate that people from any background is able to give a session, we are organizing a group of volunteers with experience giving talks to mentor people who has never given a session before, prioritizing people from underrepresented groups. If you are interested in using this help, please contact with us hola@vlctechfest.org."
+
+    - title: "Do you think we could do something else?"
+      description: "If you can think of another action we could do to improve the inclusiveness, do not doubt to share it with us: hola@vlctechfest.org."
+
+ca:
+  title: Diversitat
+  description: "En VLCTechHub portem la diversitat als nostres estatuts i sempre busquem noves iniciatives que puguem realitzar per integrar als grups infrarepressentats a la comunitat tecnológica. Açí pots encontrar les diferents iniciatives que fem per aquest festival:"
+  actions:
+    - title: "Codi de conducta"
+      description: "Per a asegurarnos que l'esdeveniment transcurreix sense incidents i on tot el mon es benvingut, incloem un codi de conducta. En el cas que siga necessari reportar una violació del codi de conducta, podeu contactar amb Emma (emma@vlctechhub.org) o David (david@vlctechhub.org)."
+
+    - title: "Mentorització de ponents"
+      description: "Per a facilitar que gent amb qualsevol transfons puga donar una cherrada, etem organitzant un grup de voluntaris amb experiencia donant cherrades per a mentoritzar a gent que mai haja donat una cherrada, prioritzant als integrants dels grups infrarrepresentats. Si t'interessa usar aquesta ajuda, per favor contacta amb David (david@vlctechhub.org)."
```

```suggestion
      description: "Per a facilitar que gent amb qualsevol rerefons puga donar una xarrada, estem organitzant un grup de voluntaris amb experiència donant xarrades per a mentoritzar a gent que mai haja donat una xarrada, prioritzant als integrants dels grups infrarrepresentats. Si t'interessa fer ús d'aquesta ajuda, per favor contacta amb David (david@vlctechhub.org)."
```

`charrada` en valenciano tiene dos formas válidas:
- `xerrada`
- `xarrada`

Lo que no es válido es escribirlo con `ch`. He optado por `xarrada` porque es la que se ha usado en la web en el resto de textos.